content/pages/comparisons/gitbook.html

```diff
@@ -0,0 +1,185 @@
+{# View this page at http://localhost:8080/comparisons/gitbook/ #}
+
+<html>
+  <head>
+    <title>Read the Docs vs. GitBook comparison</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="comparisons/gitbook" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="" />
+  </head>
+  <body>
+
+  {% import "partials/basic.html" as basic %}
+  {% import "partials/technical.html" as technical %}
+  {% import "partials/homepage.html" as homepage %}
+  {% import "partials/comparisons.html" as comparison %}
+
+  <section>
+    {% call basic.header_lead(
+    header="Read the Docs is the <em><strong>all-in-one</strong></em> solution you're looking for your documentation")
+    %}
+    <p>
+      GitBook is great. However, it lacks many features we consider <em>essentials</em> for technical teams.
+      When talking about technical documentation, <strong>Read the Docs</strong> is <em>the</em> perfect solution for it.
+    </p>
+    <p>
+      Let's compare it to know what you can expect when deploying your project's documentation to Read the Docs.
+    </p>
+    {% endcall %}
+  </section>
+
+
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call comparison.feature_table(other_img="/images/gitbook-logo.svg", other_img_alt="GitBook logo") %}
```

Comment:
I feel like normally the "things we both do" come first, and then the "things only we do" come at the bottom, but I don't feel strongly about that approach.

Reply:
Yeah, I thought about this and I wasn't sure. I left the order as it was to match GitHub Pages, but that probably doesn't make sense since they are different users/audiences and the order doesn't matter too much 👍🏼 

I will change it to match the pattern you are suggesting here.